app/core/config.py

```diff
@@ -10,7 +10,7 @@ class Settings(BaseSettings):
     # 동기 데이터베이스 연결 문자열 (Alembic 에서 사용)
     SYNC_DATABASE_URL: str
 
-    model_config = ConfigDict(env_file=".env", extra="allow")
+    model_config = ConfigDict(env_file=".env", env_file_encoding='utf-8', extra="allow")
```

Comment:
이건 다시 지우는거로 할게용. 실제 문제 잡는거랑 무관한 옵션인 것 같습니당

```suggestion
    model_config = ConfigDict(env_file=".env", extra="allow")
```

Reply:
안 지워지긴했는데 에러가 발생하는건 아니라서 resolve 처리하겠습니다